api2-war/src/main/java/eu/europeana/api2/v2/service/translate/TranslationUtils.java

```diff
@@ -228,4 +237,62 @@ public static String truncateFieldValue(String fieldValue, Integer truncateAfter
         }
         return WordUtils.abbreviate(fieldValue, truncateAfter, truncateHardLimit, TRUNCATED_INDICATOR);
     }
+
+    public static List<String> truncate(List<String> valuesToTranslate , Integer translationCharLimit, Integer translationCharTolerance) {
+        List<String> truncatedValues = new ArrayList<>();
+        if (truncationNotRequired(valuesToTranslate, translationCharLimit)) {
+            return valuesToTranslate;
+        }
```

Comment:
I'm wondering if we really need this. It looks as if we can skip it since the the for-loop should terminate normally if we don't reach the threshold?

Reply:
removed this